sys/net/gnrc/transport_layer/tcp/gnrc_tcp.c

```diff
@@ -101,35 +101,27 @@ static int _gnrc_tcp_open(gnrc_tcp_tcb_t *tcb, const uint8_t *target_addr, uint1
     /* Setup passive connection */
     if (passive) {
         /* Set Status Flags */
-        tcb->status |= STATUS_PASSIVE;
+        _common_set_status_flag(tcb, STATUS_PASSIVE);
```

Comment:
I added inline functions to make the bit fiddling in the tcbs status field less error prone.

This change just uses them everywhere the status field is changed.